nestkernel/ring_buffer.h

```diff
@@ -351,7 +352,30 @@ ListRingBuffer::get_index_( const delay d ) const
   assert( ( size_t ) idx < buffer_.size() );
   return idx;
 }
-}
+
+
+template < unsigned int num_values >
+class MultiValueRingBuffer
+{
+public:
+  MultiValueRingBuffer();
+
+  void add_value( const long idx0, const long idx1, const double val );
+
+  std::array< double, num_values >& get_values( const long idx0 );
+
+  void clear();
+
+  void resize();
+
+  size_t size() const;
+
+private:
+  //! Buffered data
+  std::vector< std::array< double, num_values > > buffer_;
```

Comment:
Please expand the docstring and explain outer and inner dimension.

Reply:
Added more info here.